src/main/java/core/mate/academy/service/TruckProduction.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProduction implements MachineProducer<Truck> {
```

Comment:
```suggestion
public class TruckProducer implements MachineProducer<Truck> {
```
change all production to producer